go.mod

```diff
@@ -65,6 +65,7 @@ replace (
 )
 
 require (
+	github.com/PuerkitoBio/purell v1.1.1 // indirect
```

Comment:
I think these unused dependencies spawn older go. We have similar discussion https://github.com/eclipse/che-machine-exec/pull/83#issuecomment-624731577 I think you should use a bit newer go to avoid this.

Reply:
With newer go: `go mod tidy` and you will see that newer go will remove these deps from go go.mod

Reply:
This changes is come when I build operator image in docker.